src/app/content/routes.ts

```diff
@@ -24,6 +25,10 @@ export const content: Route<Params, State> = {
     loading: () => null,
     modules: ['Content'],
   }),
+  getSearch: (_params: Params): string => REACT_APP_ARCHIVE_URL
```

Comment:
I'm not sure if this should be called `getSearch` since this is returning search url only for `REACT_APP_ARCHIVE_URL`
It feels like we should have just a new constant like `ARCHIVE_SEARCH_URL` for this
and then where you have
``` 
const search = content.getSearch && content.getSearch(reference.params);
const query = search ? `?${search}` : '';
```
you could just:
```
const query = REACT_APP_ARCHIVE_URL && ARCHIVE_SEARCH_URL;
```

also `_params` is not used here

Reply:
`search` is what the query string is called. https://developer.mozilla.org/en-US/docs/Web/API/HTMLHyperlinkElementUtils/search

Reply:
yes, but this function is not returning search query
it is returning search query if `ARCHIVE_SEARCH_URL` is a truthy value and empty string if it is not and the name suggests that it is getting search query from a current URL / provided params

Reply:
Okay, I get it. There are different routes and they can implement their own handler for `getSearch`
That makes more sense now, but I still don't know why it is returning an empty string if `REACT_APP_ARCHIVE_URL` is falsy? Shouldn't it return search query form the state?

Reply:
No, when content needs to get pulled from `archive`, we set this variable so that all the links included in the content, when they're clicked, they carry the `archive` part with them if there is one, if not, an empty string is `not adding the archive part to the links`

Reply:
Nice, thanks for explaining